src/Action/Processor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Jahuty\Action;
+
+use Jahuty\Api\Request;
+
+class Processor
+{
+    private $router;
+
+    public function process(Action $action): Request
+    {
+        return new Request(
```

Comment:
consider using a PSR17 factory to create the Request/Response : ) 

https://www.php-fig.org/psr/psr-17/

Reply:
Thanks for the heads up! I checked out the PSR-17 factories, but I couldn't match their method signatures. Their `RequestFactoryInterface`, for example, accepts a `string $method` and a `$uri`. I detect both of those from the `Action` requested by the Service. 

However, I love the idea, and I refactored/moved the `Jahuty\Action\Router` to `Jahuty\Uri\Factory` and the `Jahuty\Action\Processor` (admittedly, not a great name) to `Jahuty\Request\Factory`.